packages/amplify_datastore_plugin_interface/lib/src/types/query/query_field.dart

```diff
@@ -17,6 +17,7 @@ library query_field;
 
 import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
 import 'package:amplify_datastore_plugin_interface/src/types/models/model_field_type.dart';
+import 'package:flutter/foundation.dart';
```

Comment:
Nit: is this needed as no extra changes made within this file.

Reply:
Yeah, this is sort of confusing, but it is needed. This was added due to the changes in query_field_operators.dart. 

query_field_operators.dart is part of this file (part of 'query_field.dart';).